ckan/migration/versions/085_adjust_activity_timestamps.py

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+import datetime
+
+
+def upgrade(migrate_engine):
+    """
```

Comment:
Yes, you need to prefix all string literals with either `u` (if they contain Unicode) or `b` (if they contain raw bytes). In almost all cases, `u` is the correct choice. As you've already seen from the tests, this is also required for docstrings. See the discussion in #3049 for details.